docs/idea-plugin-guide/master.adoc

```diff
@@ -0,0 +1,46 @@
+
+:toc:
+:toclevels: 4
+:numbered:
+
+include::topics/templates/document-attributes.adoc[]
+
+:imagesdir: topics/images
+:context: idea-plugin
+:idea-plugin:
+
+= IntelliJ IDEA Guide
```

Comment:
Add ID

Reply:
Also, remember that if you change the title in the future, I need to update Pantheon.